docs/maintainers/issue_backlog.md

```diff
@@ -0,0 +1,176 @@
+6. Contributor begins work and publishes work to fork as it is being done
+  - posts it to the issue
+7. Contributor updates issue weekly with status updates, and pushes work to fork
+  - Periodic feedback provided
+  - Discussion between contributor and stakeholder occurs on issue
+8. Contributor sends PR for review
+  - Stakeholder ensures the appropriate reviewers exist
+  - Discussion and updates occur on the PR
+9. PR accepted and merged
+
+## Picking good issues for the backlog
+
+### What makes a good issue
+
+#### Encapsulated
+
+A good issue in the backlog can be implemented without modifying existing code.  This makes it much easier to
+review and gain consensus on.  Issues that require modifying large pieces of existing code are typically hard
+to accept without multiple reviewers, require a high degree of communication and require knowledge of the
+existing codebase.  This makes them bad candidates for contributors looking to pick up a piece of work on independently.
+
+Issues with good encapsulation have the following properties:
+
+- Minimal wiring or changes to existing code
+- Easy to disable / enable with a flag
+- Easy to review the contribution on its own without needing to examine other parts of the system
+- Low chance of needing to rebase or conflicting with changes made in parallel
+
+#### Consensus on work within the SIG
+
+Work described in issues in the backlog should be agreed upon by the SIG.  PRs sent for review should have the code
+reviewed not the concept.  SIG CLI needs to come up with low overhead a process for accepting proposed work.
+
+1. Create an issue for the work
+2. SIG agrees to accept work for the issue (as described) if it is completed
+3. Add issue to the issue backlog
+
+## Types of code contributions
+
+### Code documentation
+
+Documenting code is an excellent starter task.  It is easier to merge and get consensus on than writing tests or
+features.  It also provides a structured approach to learning about the system and its components. 
+For the packages that need it most, understanding the code base well enough to document it may be an involved task.
+
+- Adding doc.go files to packages that are missing them
+- Updating doc.go files that are empty placeholders
+- Adding examples of how to use types and functions to packages
+- Documenting functions with their purpose and usage
+
+### Test coverage
+
+Improving test coverage and augmenting e2e tests with integration tests is also a good candidate for 1st and
+2nd time contributors.  Writing tests for libraries requires understanding how they behave and are meant to be used. 
+Improving code coverage allows the project to move more quickly by reducing regressions issues that the SIG must field,
+and by providing a safety net for code reviewers ensuring changes don’t break existing functionality.
+
+- Write unit tests for functionality only covered by integration and e2e tests
```

Comment:
please provide link to kubernete's distinction between integration and e2e test.